### PR TITLE
🌐 Add Japanese translation for Deployment - Versions

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates deployment/index.md in Japanese.

To all reviewers. This section is only separated from #1948 (with few changes), but please review it.

### CODE REVIEW

**Review by komtaki — COMMENTED** [review]
Thank you for your careful work. 👍 
I commented one minor change.

**Comment by tokusumi** [comment]
thank you for your review 🙇

**Review by komtaki — APPROVED** [review]
Thanks : )

File: docs/ja/docs/deployment/versions.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+fastapi>=0.45.0,<0.46.0
+```
+
+これは `0.45.0` 以上、`0.46.0` 未満のバージョンを使うことを意味します。例えば、バージョン `0.45.2` は使用可能です。
+
+PoetryやPipenvなど、他のインストール管理ツールを使用している場合でも、それぞれパッケージのバージョンを指定する機能があります。
+
+## 利用可能なバージョン
+
+[Release Notes](release-notes.md){.internal-link target=_blank}で利用可能なバージョンが確認できます (最新版の確認などのため)。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/versions.md:37]:
```suggestion
[Release Notes](../release-notes.md){.internal-link target=_blank}で利用可能なバージョンが確認できます (現在の最新版の確認などのため)。
```
Add the implication of "current".

And dead link.

File: docs/ja/docs/deployment/versions.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## バージョンについて
+
+Semantic Versioningの規約に従って、`1.0.0` 未満の全てのバージョンは破壊的な変更が加わる可能性があります。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/versions.md:41]:
```suggestion
セマンティックバージョニングの規約に従って、`1.0.0` 未満の全てのバージョンは破壊的な変更が加わる可能性があります。
```
#L7 uses English term, so at least better to make them in the same representation.

File: docs/ja/docs/deployment/versions.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+例えば、バージョン `0.45.0` を使っているとしましょう。
+
+`requirements.txt` を使っているなら、以下の様にバージョンを指定できます:
+
+```txt
+fastapi==0.45.0
+```
+
+これは、バージョン `0.45.0` だけを使うことを意味します。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/versions.md:23]:
```suggestion
これは、厳密にバージョン `0.45.0` だけを使うことを意味します。
```
Add an implication of "exactly".

File: docs/ja/docs/deployment/versions.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+破壊的な変更と新機能実装は「マイナー」バージョンで加えられます。
+
+!!! tip "豆知識"
+    「マイナー」は真ん中の数字です。例えば、`0.2.3` ではマイナーバージョンは `2` です。
+
+## FastAPIのバージョンのアップグレード
+
+アプリケーションにテストを加えるべきです。
+
+**FastAPI** では非常に簡単に実現できます (Starletteのおかげで)。ドキュメントを確認して下さい: [テスト](tutorial/testing.md){.internal-link target=_blank}
```
Comment (sh0nk) [inline docs/ja/docs/deployment/versions.md:63]:
```suggestion
**FastAPI** では非常に簡単に実現できます (Starletteのおかげで)。ドキュメントを確認して下さい: [テスト](../tutorial/testing.md){.internal-link target=_blank}
```
dead link

**Review by sh0nk — COMMENTED** [review]
Added just a couple of dead link and suggestions! The others look so great.

**Comment by tokusumi** [comment]
@sh0nk Thank you for your great work!
I committed some improvement with your suggestions. Please check it again.

**Review by sh0nk — APPROVED** [review]
LGTM 👍

**Comment by tiangolo** [comment]
Thank you @tokusumi ! :clap: :clap: :rocket: 

And thanks for the reviews @sh0nk and @komtaki. :cake: :coffee:

**Pull request merged**